portfolio/src/main/webapp/style.css

```diff
@@ -164,32 +179,54 @@ a:hover {
 
 .white-button {
   color: #ede7f6;
+  border: 5px solid #ede7f6;
   border-radius: 5px;
-  border-style: solid;
-  border-color: #ede7f6;
   padding: 10px;
-  margin: 10px;
+  margin: 0px 10px;
   font-weight: bold;
+  display: inline-block;
 }
 
-a.white-button:hover {
+.white-button:hover {
   background-color: #ede7f6;
   color: #37474f;
 }
 
 .purple-button {
   color: #673ab7;
-  border: 5px solid #673ab7;
+  border: 2px solid #673ab7;
+  border-radius: 5px;
   padding: 10px;
-  margin: 10px;
+  margin: 0px 10px;
   font-weight: bold;
+  display: inline-block;
 }
 
-a.purple-button:hover {
+.purple-button:hover {
   background-color: #673ab7;
   color: #ede7f6;
 }
 
+.comment {
+  list-style: none;
+  color: #37474f;
+  background-color: white;
+  border: 1.5px solid #673ab7;
+  padding: 10px;
+  margin: 0px 0px 30px 0px;
```

Comment:
Nit: could just use margin-bottom here

Reply:
Since<p> tags have default margins, I specified the 0px to override the defaults.

Reply:
Oh, I see!